cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -108,6 +108,25 @@ spec:
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
         {{ end }}
+        {{ if .EnableVolumeReplicationSidecar }}
+        - name: volume-replication
+          image: {{ .VolumeReplicationImage }}
+          args :
+            - "--metrics-bind-address=0"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--drivername={{ .DriverNamePrefix }}rbd.csi.ceph.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--rpc-timeout=150s"
+            - "--health-probe-bind-address=:9998"
+            - "--leader-elect=false"
+          env:
+            - name: ADDRESS
```

Comment:
Why use an env var? If we always want to use the socket and it doesn't need to be configurable, it could be specified directly in the args.

Reply:
following the pattern used in other places. Should I continue the pattern or directly put it in the args?

Reply:
Oh, same pattern sounds good